code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -120,7 +105,7 @@
 		if(M.buckled)
 			continue
 
-		moveMob(M, src)
+		M.forceMove(src)

```

Comment:
Mhm, wouldn't this also happen on /tg/ in that case? This change was ported directly from them, in order to make the stasis pod for the anomalous crystal work.